### PR TITLE
fix: MapPopup render problem

### LINKED ISSUE #423: Broken profile section styles
image

image

image

### PR DESCRIPTION
**Pull request by JustinCast**
# Fix MapPopup render problem
This PR provides a fix for the render problem when calling the component
# How does this should be tested?
- `make run`
- Login as sponsor or lifebank and go to the profile section
- Check if there is anything unusual
## Linked issues
closes #423

### CODE REVIEW

File: webapp/src/components/MapShowOneLocation/index.js [hunk 1]
```diff
@@ -56,9 +56,15 @@ function MapShowOneLocation({ markerLocation, accountProp, ...props }) {
         const popupNode = document.createElement('div')
 
         ReactDOM.render(
-          <MapPopup id={id} info={info} username={user.username.replaceAll(' ', '-')} account={account} />,
+          <MapPopup
+            id={id}
+            info={info}
+            username={user.username}
```
Comment (AngeloCG97) [inline webapp/src/components/MapShowOneLocation/index.js:62]: in this line a replace all necessary was eliminated

**Review by AngeloCG97 — CHANGES_REQUESTED** [review]

**Comment by JeanVegaD** [comment]
there seem to be layout issues with the popup

image

**Comment by JeanVegaD** [comment]
check code factor suggestions

**Comment by JustinCast** [comment]
> there seem to be layout issues with the popup
> 
> image

That is more concerning with the design than with the fix itself

**Review by AngeloCG97 — CHANGES_REQUESTED** [review]
Design is broke

image

File: hapi/src/api/account.api.js [hunk 2]
```diff
@@ -68,11 +68,11 @@ const create = async ({ role, email, name, secret }) => {
...
   await historyApi.insert(transaction)
 
-  try {
-    mailApi.sendVerificationCode(email, verification_code)
-  } catch (error) {
-    console.log(error)
-  }
+  // try {
+  //   mailApi.sendVerificationCode(email, verification_code)
+  // } catch (error) {
```
Comment (AngeloCG97) [inline hapi/src/api/account.api.js:73]: delete comments before to push

**Review by AngeloCG97 — CHANGES_REQUESTED** [review]

**Review by JeanVegaD — CHANGES_REQUESTED** [review]
I think it looks very small

image

**Review by AngeloCG97 — APPROVED** [review]

**Review by JeanVegaD — APPROVED** [review]

File: webapp/src/components/MapPopup/index.js [hunk 3]
```diff
@@ -1,51 +1,93 @@
 import React, { useEffect, useState } from 'react'
 import PropTypes from 'prop-types'
 import Button from '@material-ui/core/Button'
-import { makeStyles } from '@material-ui/styles'
+import { makeStyles, ThemeProvider } from '@material-ui/styles'
```
Comment (JustinCast) [inline webapp/src/components/MapPopup/index.js]: Theme provider seems not to be used

File: webapp/src/components/MapPopup/index.js [hunk 4]
```diff
@@ -149,56 +195,45 @@ const MapPopup = ({ id, info, username }) => {
...
         </div>
       )}
-      <div
-        style={{ alignSelf: 'center', marginBottom: '3px' }}
-        className={classes.row}
-      >
-        <span className={classes.boldText}>{t('signup.address')}: &nbsp;</span>
+      <div className={classes.row}>
+        <h2 className={classes.boldText} >
+          {t('signup.address')}:{' '}
```
Comment (JustinCast) [inline webapp/src/components/MapPopup/index.js:216]: Curly braces here

**Pull request merged**